### ISSUE TITLE
starting docker fail with java project

### ISSUE THREAD

**Issue by XuhaoZhao** [agentic-labs/lsproxy#78]
PS C:\Users\Administrator\Desktop\lsproxy> docker run -p 4444:4444 -v /c/Users/Administrator/Desktop/spring-cloud-consul:/mnt/workspace agenticlabs/lsproxy:latest
Starting...
[2024-11-21T05:33:25Z INFO  lsproxy] Logger initialized
WARNING: Using incubator modules: jdk.incubator.vector, jdk.incubator.foreign
Nov 21, 2024 5:33:32 AM org.apache.aries.spifly.BaseActivator log
INFO: Registered provider ch.qos.logback.classic.servlet.LogbackServletContainerInitializer of service jakarta.servlet.ServletContainerInitializer in bundle ch.qos.logback.classic
Nov 21, 2024 5:33:32 AM org.apache.aries.spifly.BaseActivator log
INFO: Registered provider ch.qos.logback.classic.spi.LogbackServiceProvider of service org.slf4j.spi.SLF4JServiceProvider in bundle ch.qos.logback.classic
Error: Custom { kind: Other, error: "deadline has elapsed" }

**Comment by maks-ivanov** [comment]
Hey, thanks for flagging!
I was unable to reproduce it on my machines, could you share more about your system?
This might be a simple timeout during java langserver startup, looking into it

**Comment by XuhaoZhao** [comment]
yes, it is problem of machine,  i came into it on the win10 with docker,  and it work well on my mac mini
